votingpool/pool.go

```diff
@@ -387,11 +467,15 @@ func validateAndDecryptKeys(rawPubKeys, rawPrivKeys [][]byte, vp *Pool) (pubKeys
 // seriesLookup map with them. If there are any private extended keys for
 // a series, it will also ensure they have a matching extended public key
 // in that series.
-func (vp *Pool) LoadAllSeries() error {
+//
+// This method must be called with the Pool's manager unlocked.
+// FIXME: We should be able to get rid of this (and loadAllSeries/seriesLookup)
```

Comment:
This one was just added when addressing one of your comments here, so yes I'd rather leave it for later.